css/style.css

```
@@ -1,3 +1,62 @@
-h1 {
-    color: blue;
+body {
+    margin: 0;
+    font-family: Noto Sans JP;
```

`margin: 0` は リセットCSSで定義されているから不要になります🙏
あと、Notoを読めなかったときのフォールバックとして `serif` を追加ください（font-familyの指定の仕方と意味は後ほど詳細説明します）
```suggestion
    font-family: "Noto Sans JP", sans-serif;;
```